pkg/controller/pod_volume_backup_controller.go

```diff
@@ -300,19 +307,14 @@ func (c *podVolumeBackupController) fail(req *velerov1api.PodVolumeBackup, msg s
 	if _, err := c.patchPodVolumeBackup(req, func(r *velerov1api.PodVolumeBackup) {
 		r.Status.Phase = velerov1api.PodVolumeBackupPhaseFailed
 		r.Status.Message = msg
+		r.Status.CompletionTimestamp.Time = c.clock.Now()
 	}); err != nil {
 		log.WithError(err).Error("Error setting phase to Failed")
 		return err
 	}
 	return nil
 }
 
-func updatePhaseFunc(phase velerov1api.PodVolumeBackupPhase) func(r *velerov1api.PodVolumeBackup) {
```

Comment:
this was only being used in one place so I thought I'd remove it and just use an anonymous function to update both the phase and StartTimestamp field

Reply:
👍